endpoints/private/rulesfn/uri.go

```diff
@@ -65,17 +65,17 @@ func ParseURL(input string) *URL {
 	// be reverted since the returned URL will be used in string builders.
 	authority := strings.ReplaceAll(u.Host, "%", "%25")
 
-	var isIP bool
+	var isIp bool
```

Comment:
Why did we change this? I'm pretty sure go styling prefers the original state.

Reply:
its because the models only case the first letter of acronyms. the generator takes that casing by default, and there would have to be special logic that capitalizes all letters of acronyms. i couldnt readily find a utility in the Smithy utils that does this. and this seems like something we should use a library for and i wasnt sure if our org had any open-source policy on what 3P tools we can include in our repositories, do you know of any restrictions? all that said, i can make this work though if we need to.